content/docs/1.7.0/snapshots-and-backups/scheduling-backups-and-snapshots.md

```diff
@@ -14,6 +14,8 @@ You can configure,
 - The number of backups or snapshots to retain
 - The number of jobs to run concurrently
 - Any labels that should be applied to the backup or snapshot
+- Parameters that should be applied to the backup or snapshot
```

Comment:
Current parameters field only show up when user select `backup` task. Non for snapshot.

Reply:
fixed. thanks